libraries/ms-helper/inc/exported_enums.h

```diff
@@ -200,11 +200,15 @@ typedef enum EESolarFault {
   // Fan failure detected. No data.
   EE_SOLAR_FAULT_FAN_FAIL,
 
-  // The drv120 relay has signaled that overtemp/undervolt lockout conditions have been triggered
-  EE_SOLAR_FAULT_DRV120,
-
-  // The drv120 relay has not opened
-  EE_SOLAR_RELAY_OPEN_ERROR,
-
   NUM_EE_SOLAR_FAULTS,
 } EESolarFault;
+typedef enum EESolarRelayOpenErrorReason {
```

Comment:
Nit: blank line between the enums please.

Reply:
done